packages/kotti-ui/source/kotti-field-inline-edit/KtFieldInlineEdit.vue

```diff
@@ -306,6 +306,8 @@ export default defineComponent({
 
 .kt-field-inline-edit {
 	// slot that wraps <input /> & <EditIcon/>
+
+	/* stylelint-disable-next-line selector-class-pattern */
```

Comment:
the only non-fixed classes here were kt-field classes because there's another issue to handle this where we will turn kt-field classes into templates